en/components/combo.md

```diff
@@ -6,105 +6,88 @@ _keywords: Design Systems, Design Systems UX, UI kit, Figma, Figma to Angular, E
 
-Every experienced designer uses constraints wisely to limit the user input and avoid invalid states, hence the availability of validation styles. Through the available validation styles, the Combo is consistent with the standard [Input](input.md) and equipped for sophisticated designs that display success, warning, and error visuals.
+| Idle | Focused | Filled | 
+| ------- | ------- | ------- |
+| <img class="responsive-img" src="../images/combo_idle.png" srcset="../images/combo_idle@2x.png 2x" /> | <img class="responsive-img" src="../images/combo_focused.png" srcset="../images/combo_focused@2x.png 2x" /> | <img class="responsive-img" src="../images/combo_filled.png" srcset="../images/combo_filled@2x.png 2x" /> |
 
-<img class="responsive-img" src="../images/combo_success.png" srcset="../images/combo_success@2x.png 2x" />
-`success`
-<img class="responsive-img" src="../images/combo_warning.png" srcset="../images/combo_warning@2x.png 2x" />
-`warn`
-<img class="responsive-img" src="../images/combo_error.png" srcset="../images/combo_error@2x.png 2x" />
-`error`
+Every experienced designer uses constraints wisely to limit the user input and avoid invalid states, hence the availability of validation styles. Through the available validation styles, the Combo Inputs is consistent with the standard [Input](input.md) and equipped for sophisticated designs that display success, warning, and error visuals.
 
-<div class="divider--half"></div>
-<div class="divider--half"></div>
+| Success | Warning | Error | 
+| ------- | ------- | ------- |
+| <img class="responsive-img" src="../images/combo_success.png" srcset="../images/combo_success@2x.png 2x" /> | <img class="responsive-img" src="../images/combo_warning.png" srcset="../images/combo_warning@2x.png 2x" /> | <img class="responsive-img" src="../images/combo_error.png" srcset="../images/combo_error@2x.png 2x" /> |
 
 In Sketch, the states and validation states are achieved with `Symbol Overrides`, while in Adobe XD we are using the `Component States` paradigm to let you easily switch between them. In Figma changing the states is achieved via the properties panel.
 
 > [!Note]
-> The Combo input has only one layout option, but this override has been preserved in Sketch to ensure consistency with the standard [Input](input.md). In Figma on the other hand we have a separate base component called `.Combo Input`, which is used explicitly in the Combo component.
+> The Combo Input has only one layout option, but this override has been preserved in Sketch to ensure consistency with the standard [Input](input.md). In Figma we have a separate base component called `.Combo Input`, which is explicitly used in the Combo component for both Single and Multiple selection types. The Combo Input has a Prefix container which can be switched **on** and **off**.
```

Comment:
In Sketch the Combo Input has only one layout option, however, the overrides have been preserved, in order to ensure consistency with the standard [Input] (input.md).